apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -852,97 +370,220 @@ const PoolCandidatesTable = ({
-          id: "z0QI6A",
-          description: "Title for the admin pool candidates table",
-        })}
-      </h2>
-      <TableHeader
-        columns={columns}
-        filterComponent={
-          <PoolCandidateTableFilterDialog
-            onSubmit={handlePoolCandidateFilterSubmit}
-            initialFilters={initialFilters}
-          />
-        }
-        onSearchChange={(
-          term: string | undefined,
-          type: string | undefined,
-        ) => {
-          handleSearchStateChange({
-            term,
-            type,
-          });
-        }}
-        initialSearchState={searchState}
-        searchBy={[
-          {
-            label: intl.formatMessage({
-              defaultMessage: "Name",
-              id: "36k+Da",
-              description: "Label for user table search dropdown (name).",
-            }),
-            value: "name",
+  const columns = [
+    columnHelper.accessor(
+      ({ poolCandidate: { status } }) =>
+        intl.formatMessage(
+          status ? getPoolCandidateStatus(status) : commonMessages.notFound,
+        ),
+      {
+        id: "status",
+        sortingFn: normalizedText,
+        header: intl.formatMessage(tableMessages.status),
+        cell: ({
+          row: {
+            original: { poolCandidate },
+          },
+        }) => statusAccessorNew(poolCandidate.status, intl),
+      },
+    ),
+    columnHelper.accessor(
+      ({ poolCandidate: { user } }) =>
+        intl.formatMessage(
+          user.priorityWeight
+            ? getPoolCandidatePriorities(user.priorityWeight)
+            : commonMessages.notFound,
+        ),
+      {
+        id: "priority",
+        header: intl.formatMessage(tableMessages.category),
+        cell: ({
+          row: {
+            original: {
+              poolCandidate: { user },
+            },
+          },
+        }) => priorityAccessorNew(user.priorityWeight, intl),
+      },
+    ),
+    columnHelper.display({
+      // TODO: Should this be an accessor instead of a display type?
+      id: "candidacyStatus",
+      header: intl.formatMessage(tableMessages.candidacyStatus),
+      cell: ({
+        row: {
+          original: { poolCandidate },
+        },
+      }) => candidacyStatusAccessorNew(poolCandidate.suspendedAt, intl),
+    }),
+    columnHelper.display({
+      id: "view",
+      header: intl.formatMessage(tableMessages.view),
+      cell: ({
+        row: {
+          original: { poolCandidate },
+        },
+      }) => viewPoolCandidateAccessor(poolCandidate, paths, intl),
+    }),
+    columnHelper.accessor(
+      ({ poolCandidate: { user } }) => userNameAccessor(user),
+      {
+        id: "candidateName",
+        header: intl.formatMessage(tableMessages.candidateName),
+        sortingFn: normalizedText,
+        cell: ({
+          row: {
+            original: {
+              poolCandidate: { user },
+            },
+          },
+        }) => `${user?.firstName} ${user?.lastName}`,
```

Comment:
This looks like it returns the same as the accessor. If a cell definition does not exist, it will render the result of the accessor by default so not need for the cell here 👍

Reply:
Fixed by [95daca8](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8360/commits/95daca8101e574241aa2f72decff7b9cc7c12cdd)